packages/kit/src/core/env.js

```diff
@@ -23,6 +24,14 @@ export function create_module(id, env) {
 	return GENERATED_COMMENT + declarations.join('\n\n');
 }
 
+/**
+ * @param {'public' | 'private'} type
+ * @returns {string}
+ */
```

Comment:
I tend to think explicit `@return` annotations are unnecessary in most cases, especially small helper functions like this one — getting rid allows us to collapse the JSDoc annotation onto a single line in a lot of cases

```suggestion
/** @param {'public' | 'private'} type */
```